source/flight_software/propulsion_system.rst

```diff
@@ -0,0 +1,56 @@
+This file describes the Propulsion System driver. 
+
+PropulsionSystem class defines the interface for the propulsion system, which 
+ consists of an inner tank, Tank1, and the (outer) thrust tank, Tank2. 
+ This driver provides functionality for opening and closing valves on both tanks,
+ setting and enforcing the thrust schedule 
+ 
+ It consists of a static Tank1 (inner tank) and Tank2 (thrust tank) data objects. 
+ 
+ Specifications:
+ Tank1 valves are at numbered 0, 1
+ Tank2 valves are at numbered 0, 1, 2, 3
+ mandatory_wait_time is the duration of time after opening a valve before 
+ we can open another valve. Tank1 10*1000 ms. Tank2 is 3ms
```

Comment:
I think you could have a whole section on wait time and explaining _why_ we have wait times.